qualtran/bloqs/basic_gates/swap.py

```diff
@@ -159,10 +159,9 @@ def _t_complexity_(self) -> 'TComplexity':
         """The t complexity.
 
         References:
-            [An algorithm for the T-count](https://arxiv.org/abs/1308.4134). Gosset et. al. 2013.
-            Figure 5.2.
+            [An algorithm for the T-count](https://arxiv.org/abs/1308.4134).
+            Gosset et. al. 2013. Figure 5.2.
         """
-        # https://arxiv.org/abs/1308.4134
         return TComplexity(t=7, clifford=10)
```

Comment:
Can you actually add the decomposition that was deleted from `_fredkin` here? 

```
    yield [cirq.CNOT(t2, t1)]
    yield [cirq.CNOT(c, t1), cirq.H(t2)]
    yield [cirq.T(c), cirq.T(t1) ** -1, cirq.T(t2)]
    yield [cirq.CNOT(t2, t1)]
    yield [cirq.CNOT(c, t2), cirq.T(t1)]
    yield [cirq.CNOT(c, t1), cirq.T(t2) ** -1]
    yield [cirq.T(t1) ** -1, cirq.CNOT(c, t2)]
    yield [cirq.CNOT(t2, t1)]
    yield [cirq.T(t1), cirq.H(t2)]
    yield [cirq.CNOT(t2, t1)]
```

Reply:
added